plugins/modules/remote_management/redfish/redfish_config.py

```diff
@@ -100,6 +100,18 @@
     type: bool
     default: false
     version_added: 3.7.0
+  hostinterface_config:
+    required: false
+    description:
+      - Setting dict of HostInterface on OOB controller.
+    type: dict
+    version_added: '4.1.0'
+  hostinterface_id:
+    required: false
+    description:
+      - Redfish HostInterface instance ID if multiple HostInterfaces are present.
+    type: str
+    version_added: '3.9.0'
```

Comment:
```suggestion
    version_added: '4.1.0'
```
This one is still missing.